lib/parallel_tests/tasks.rb

```diff
@@ -166,6 +194,22 @@ def rails_61_or_greater?
     )
   end
 
+  # drop individual databases in multi DB setups
+  ParallelTests::Tasks.for_each_database do |name|
+    desc "Drop test #{name} database via db:drop:#{name} --> parallel:drop:#{name}[num_cpus]"
+    task "drop:#{name}".to_sym, :count do |_, args|
+      ParallelTests::Tasks.run_in_parallel(
+        [
+          $0,
+          "db:drop:#{name}",
+          "RAILS_ENV=#{ParallelTests::Tasks.rails_env}",
+          "DISABLE_DATABASE_ENVIRONMENT_CHECK=1"
+        ],
+        args
+      )
+    end
+  end
```

Comment:
can we reduce the copy paste by making for_each_database also return `nil` for the base db ?

Reply:
that sounds good :)